src/components/Heading.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { fontSizes } from '../theme'
 
-const Heading = ({ level, children, size }) => {
+const Heading = ({ level = 1, children, size = 'large' }) => {
```

Comment:
I don't think there's a logical default for level and size, in the place that we define the replacements we can manually create them.
```suggestion
const Heading = ({ level, children, size }) => {
```